src/import/pull-job.c

```diff
@@ -176,7 +176,7 @@ void pull_job_curl_on_finished(CurlGlue *g, CURL *curl, CURLcode result) {
 
         if (j->disk_fd >= 0 && j->allow_sparse) {
                 /* Make sure the file size is right, in case the file was
-                 * sparse and we just seeked for the last part */
+                 * sparse and we just sought for the last part */
```

Comment:
It's 'seeked' because we call fseek(). I think it's better to keep this.

Reply:
Yeah, I was wondering about that as well. I usually do `seek'd` here, but I'll keep `seeked` unless you prefer the apostrophe.